helper/strutil/strutil.go

```diff
@@ -301,6 +301,22 @@ func EquivalentSlices(a, b []string) bool {
 	return true
 }
 
+// EqualStringMaps tests whether two map[string]string objects are equal
+func EqualStringMaps(a, b map[string]string) bool {
```

Comment:
I have a silly question. Why use this instead of `reflect.DeepEqual`? When I change this method to simply use that, the tests still pass. Ex.
```
// EqualStringMaps tests whether two map[string]string objects are equal
func EqualStringMaps(a, b map[string]string) bool {
	return reflect.DeepEqual(a, b)
}
```
It seems like a lot of the methods in this package might be similar in that sense. I may be missing something about what this package is used for.

Reply:
The reason in this case is the cost of reflection. To test that assumption, I just ran a benchmark as a quick experiment and the same function using `reflect.DeepEqual()` is taking ~10x the time as the PR version. If it was just in the auth function, that would probably not be a major factor. But since I'm proposing adding a library function for general use (since `map[string]string` seems a somewhat common structure), I wanted to avoid the perf hit of `reflect`.

Reply:
Awesome, that's what I figured, but thought I'd ask to be sure.

Reply:
You might want to update the reasoning behind this utility as a comment for this function so people can choose whether or not to use this depending on the usecase.